src/puppetlabs/rbac_client/services/activity.clj

```diff
@@ -50,8 +50,15 @@
   (init [this context]
     (let [api-url (get-in-config [:activity-consumer :api-url])
           ssl-config (get-in-config [:global :certs])
-          route-limit {:max-connections-per-route 20}
-          client (create-client (merge route-limit ssl-config))]
+          connection-limits {:max-connections-per-route (get-in-config [:activity-consumer :max-connections-per-route]
+                                                                       (get-in-config [:activity-consumer :max-connections-per-route] 20))
+                             :max-connections-total (get-in-config [:activity-consumer :max-connections-per-route]
+                                                                   (get-in-config [:activity-consumer :max-connections-total] 20))}
+          authenticated-connection-limits {:max-connections-per-route (get-in-config [:activity-consumer :max-connections-per-route-auth]
+                                                                                     (:max-connections-per-route connection-limits))
+                                           :max-connections-total (get-in-config [:activity-consumer :max-connections-total-auth]
+                                                                                 (:max-connections-total connection-limits))}
+          client (create-client (merge authenticated-connection-limits ssl-config))]
```

Comment:
We should also add some logging similar to https://github.com/puppetlabs/orchestrator/blob/fc882562ab6430d5a7ba87e38089871613465784/src/puppetlabs/orchestrator/service.clj#L407-L408 for each service.

Reply:
I split out the logging into a subtask, so I was planning on doing that in a different PR. Whichever you think makes more sense

Reply:
Sounds great.